mappings/net/minecraft/recipe/Recipe.mapping

```diff
@@ -1,18 +1,40 @@
 CLASS net/minecraft/class_1860 net/minecraft/recipe/Recipe
-	METHOD method_17447 getRecipeKindIcon ()Lnet/minecraft/class_1799;
+	METHOD method_17447 getIconStack ()Lnet/minecraft/class_1799;
+		COMMENT The Item stack to be displayed on the recipe book tab containing this recipe.
```

Comment:
I don't think this is actually used for tabs in the recipe book, but I do know it's used in toasts (see the `RecipeToast` class):

```suggestion
		COMMENT Creates the stack that is displayed on the recipe book tab containing this recipe, and on a toast when a recipe of this type is unlocked.
```

```suggestion
		COMMENT Creates the stack that is displayed on a toast when a recipe of this type is unlocked.
```